### PR TITLE
add some notable contributors as potential spirit names in occultism

### PR DESCRIPTION
**Pull request by atlgeek007**

### CODE REVIEW

**Comment by NielsPilgaard** [comment]
Thanks for the PR! 

To be frank, I think putting your own name in the list of notable contributors is stretching it a bit, seeing as this is your first PR to E8 :P

**Comment by atlgeek007** [comment]
hehe, that was largely just me putting it in for vanity, and for some of the stuff I put in for the e6 wiki, I'll remove it :P

File: defaultconfigs/occultism-server.toml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  ritualDurationMultiplier = 1.0
+  #By default spirit names are generated at random from lists of possible syllables. If you instead want to specify the possible spirit names directly, configure a list of values here.
+  possibleSpiritNames = [
+                          "Kanzaji",
+                          "Motoko",
+                          "Darkere",
+                          "Discomanco",
+                          "TheBoo",
+                          "TemperedFool",
+                          "Ridinisaurus"
```
Comment (Darkere) [inline defaultconfigs/occultism-server.toml:73]: typo

**Comment by Kanzaji** [comment]
Yay I wasn't removed from that list 😂

**Review by NielsPilgaard — APPROVED** [review]
Solid start, thanks 👍

**Pull request merged**